src/main/app/forms/validation/validators/postCodeNotInScotlandOrNI.ts

```diff
@@ -0,0 +1,27 @@
+import { ValidationArguments, ValidatorConstraint, ValidatorConstraintInterface } from "@hmcts/class-validator"
+import { ValidationErrors } from "forms/models/address"
+
+@ValidatorConstraint({ name: 'postcodeNotInScotlandOrNI', async: false })
+export class PostcodeNotInScotlandOrNIValidator implements ValidatorConstraintInterface {
```

Comment:
Please add unit tests for this validator. Example here: https://github.com/hmcts/cmc-validators/blob/master/src/test/isValidPostcode.spec.ts

Reply:
added